lib/krikri/parser.rb

```diff
@@ -162,6 +187,18 @@ def field(*args)
       end
 
       ##
+      # Accesses the union of multiple specified fields.
+      #
+      # @return [ValueArray] an array containing the nodes available in the
+      # given fields.
+      def fields(*args)
```

Comment:
Yes. 

And `records.fields([:foo, :bar], [:foo, :bar, :baz])` should return the union of the fields `:bar` in `:foo` and `:baz` in `:bar`.